### ISSUE TITLE
addon: facebook

### ISSUE THREAD

**Issue by karbovskibg** [friendica/friendica#4778]
is there some addon that connect with facebook i see there is facebook for friendico but it is depreceated

**Comment by karbovskibg** [comment]
include petovan i see you.

**Comment by MrPetovan** [comment]
Hi, a while ago Facebook changed their policy regarding third-party access to their API, which makes it very hard for us to offer a Facebook connectivity addon. As such, the legacy addon has been deprecated and no work is planned on a new one.

**Comment by AlfredSK** [comment]
There is no bidirectional connector. (read MrPetovan) :-)
But you can post to FB via the Buffer add-on.

**Comment by karbovskibg** [comment]
lfredSK in friendica you are k9 i write you other day but you didn't comment nothing. lfredSK that is second issue. Posts are going to all accounts that are enabled by default:

    Yourname in facebook Yoursurename in facebook (Facebook)

So this is serious issue if you have more than 2xx user in friendica and you enabe buffer all 2xx can use your facebook account. So it is necessary to make individual facebook account, with all this client shits and secret id they much easly can hack everything. Just in case, do someone know how to get from stock rom device tree " /lineage/device/nameofsmartphone/nameofsmartphone/ and /lineage/vendor/nameofsmartphone/nameofsmartphone/

**Comment by MrPetovan** [comment]
Isn't Buffer configuration user-specific? In this case, each user uses their own Facebook account from Buffer.

**Comment by karbovskibg** [comment]
No... in this case my brother enter from user account he use my facebook, how i say if 2xx users use buffer they will use facebook but in this case my brother try buffer and it didn't work how was wait. Function of buffer is not working, when my brother post something it don't display in facebook and vice verce " when my brother post from my facebook it don't appear in friendica.

**Comment by karbovskibg** [comment]
and addon twitter i didn't test it and my brother don't have twitter, but in twitter addon it ask separattely every user to enter his twitter account this is normal not like buffer 2xx user your personal facebook account. shame.

**Comment by annando** [comment]
The buffer settings are - of course - individual settings that are stored in the personal configuration settings. Don't mix this with the OAuth registration that has to be done in the administration.

And to clarify: Buffer is a "posting only" connector. There is no way to import posts from Facebook to Friendica.

**Comment by karbovskibg** [comment]
annando my brother condigure oath, and now he can use my facebook incclude the users who are registered in server
